src/api/routes.ts

```diff
@@ -104,6 +104,8 @@ export const buildGetMembersBy = (emails: string[]) =>
 export const buildGetMember = (id: UUID) => `${MEMBERS_ROUTE}/${id}`;
 export const buildGetMembersRoute = (ids: UUID[]) =>
   `${MEMBERS_ROUTE}?${qs.stringify({ id: ids }, { arrayFormat: 'repeat' })}`;
+export const buildGetMemberStorage = () => `${MEMBERS_ROUTE}/current/storage`;
```

Comment:
Question: Is it planned to have other info under the `/current/` route ?

Reply:
Maybe? the safe?